examples/plot_01_getting_started.py

```diff
@@ -1,24 +1,25 @@
 """
-=================================
-1) Getting started with ``skore``
-=================================
+.. _example_getting_started:
 
-This example runs the :ref:`getting_started` guide.
+===============
+Getting started
+===============
 
-``skore`` UI
-------------
+This example runs the :ref:`getting_started` guide and adds examples a more
+types that can be stored.
 
-This section provides a quick start to the ``skore`` UI, an open-source package that aims to enable data scientists to:
+``skore``'s Project and UI
+--------------------------
 
-#. Store objects of different types from their Python code: python lists, ``scikit-learn`` fitted pipelines, ``plotly`` figures, and more.
+This section provides a quick start to skore's Project and UI, an open-source package that aims to enable data scientists to:
+
+#. Store objects of different types from their Python code: python lists, scikit-learn fitted pipelines, plotly figures, and more.
 #. Track and visualize these stored objects on a user-friendly dashboard.
-#. Export the dashboard to a HTML file.
 
 Initialize a Project and launch the UI
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-From your shell, initialize a skore project, here named ``my_project_gs``, that
-will be in your current working directory:
+From your shell, initialize a skore project, here named ``my_project_gs``:
```

Comment:
can we change everywhere `my_project_gs` to have only one project name over the whole doc? So people can copy/paste from anywhere.

Reply:
The examples are actually ran to build the doc so it will create skore projects on the folder, I am afraid on overlapping item names